index.html

```diff
@@ -1539,8 +1511,9 @@ <h4>Examples</h4>
          41f1328c-0571-4e71-8be8-e65bc148281a
       &lt;/dc:identifier>
       &lt;dc:language>en-Brai-US&lt;/dc:language>
+      &lt;dc:date>2014-12-11T09:12:52+00:00&lt;/dc:date>
       &lt;meta property="dcterms:dateCopyrighted">
-         2024
+         2004
```

Comment:
why did this change from 2024 to 2004?

Reply:
The copyright date is of the original work. World Cultures and Geography was published in 2004, so that's the correct year.

The dc:date of 2014 is when the epub 3 sample this metadata came from was created. I think we can update that to 2025 since we're mocking up data for an ebraille version of it.

Reply:
Should dateCopyrighted be recommended and not required?

What if you're creating an original braille work? What would you set this field to in that case?